ckan/views/user.py

```diff
@@ -28,6 +28,12 @@
 user = Blueprint(u'user', __name__, url_prefix=u'/user')
 
 
+@user.errorhandler(403)
+def error_handler(e):
+    extra_vars = {u'code': e.code, u'content': e.description}
+    return base.render(u'403.html', extra_vars)
```

Comment:
This will return a 200 status, which is not what we want as we need to keep the original exception status code. To return the status code you can do the following ([docs](http://flask.pocoo.org/docs/1.0/quickstart/#about-responses)):

```python
return base.render(u'403.html', extra_vars), e.code
```